src/main/scala/it/pagopa/interop/attributeregistryprocess/api/impl/AttributeRegistryApiServiceImpl.scala

```diff
@@ -176,15 +176,17 @@ final case class AttributeRegistryApiServiceImpl(
 
   }
 
-  private def getAll[T](limit: Int)(get: (Int, Int) => Future[Seq[T]]): Future[Seq[T]] = {
+  private def getAllPages[T](limit: Int)(get: (Int, Int) => Future[Seq[T]]): Future[Seq[T]] = getAll(limit, 1)(get)
+
+  private def getAll[T](limit: Int, firstOffsetValue: Int = 0)(get: (Int, Int) => Future[Seq[T]]): Future[Seq[T]] = {
```

Comment:
I'll remove this default as it doesn't seem to be needed

Reply:
I introduce this approach to keep the `getAll` function working on the offset.
Offset should take 0 as the first value, so I put it as the default.
Before this change, 0 was hard coded as input to the internal `go` function.

In general, I want to have an explicit function for pages without duplicate code.